test/html-to-react.js

```diff
@@ -13,6 +16,10 @@ describe('html-to-react', () => {
       });
     });
 
+    it('has default ES Module defined', () => {
+      assert.notEqual(ESParser['default'], undefined);
```

Comment:
Nit: you can remove lines 6-8 and update test to:
```js
assert.strictEqual(parse.default, parse);
```

Reply:
All right!